src/backends/meta-egl.c

```diff
@@ -656,23 +656,24 @@ meta_egl_stream_consumer_output (MetaEgl          *egl,
   return TRUE;
 }
 
-gboolean
+EGLint
```

Comment:
Sure thing. Will you rebase wip/egldevice on top of wip/meta-egl-error? Or do you want me to just cherry-pick that commit?